ui/pages/send/send-content/send-amount-row/send-amount-row.component.js

```diff
@@ -118,15 +42,15 @@ export default class SendAmountRow extends Component {
   }
 
   render() {
-    const { gasTotal, inError } = this.props;
+    const { inError } = this.props;
 
     return (
       <SendRowWrapper
         label={`${this.context.t('amount')}:`}
         showError={inError}
         errorType="amount"
       >
-        {gasTotal && <AmountMaxButton inError={inError} />}
```

Comment:
It looks like we're no longer hiding this button while waiting for the gas data to load. Perhaps we could bring back this conditional, with a check for the send stage instead of checking `gasTotal`? Or is this handled already elsewhere?

Reply:
The button is rendered but disabled if gas is loading.

Reply:
Well, it's disabled in that it won't work. But it's still shown on-screen, and doesn't have the `disabled` attribute set. It's possible this was done so that users wouldn't click it while gas was loading, thinking it was broken.

It might be better to either restore this logic and hide it while gas is loading, or set the `disabled` prop on the button while gas is loading, so that users don't expect clicking it to do something.

Reply:
https://github.com/MetaMask/metamask-extension/pull/10965/commits/7209ec6a38f2da6369804c8a629da195179e32da